src/expr.cpp

```diff
@@ -400,16 +400,11 @@ Expr *Expr::PartialWrt(hParam p) const {
     ssassert(false, "Unexpected operation");
 }
 
-void Expr::ParamsUsedList(std::vector<hParam> *list) const {
```

Comment:
Have you really tested it? There is no "large" param sets, linear search is the best suitable for this kind of data.